src/components/TableProperty/TableProperty.tsx

```diff
@@ -382,6 +387,118 @@ const TableProperty = ({
     }
   }
 
+  function handleCountdownTimer(property: Properties) {
```

Comment:
Add comment to explain complex logic for this function

Reply:
fixed